cli/src/factory_ops/boxing/prov_tasks

```diff
@@ -105,6 +106,10 @@ function create_unboxing_user {
     useradd --base-dir /tmp --inactive 2 --groups wheel --shell /usr/bin/bash --password $(openssl passwd -1 ${_secret}) cortxub
     passwd -e $_user
 
+    ###### Lyve Rack Serial Number ######
+    echo -n "INFO: Getting Lyve Rack serial numbers " | tee -a ${LOG_FILE}
+    [[ -f /opt/seagate/lr-serial-number ]] || _lr_serial=$(cat /opt/seagate/lr-serial-number) && echo "WARNING: Lyve Rack serial number has not been provided" 
```

Comment:
Should be:
`[[ -f /opt/seagate/lr-serial-number ]] && _lr_serial=$(cat /opt/seagate/lr-serial-number) || echo "WARNING: Lyve Rack serial number has not been provided"`